tests/test_blog.py

```
@@ -0,0 +1,391 @@
+            "metadata": {
+                "total_clips": 1,
+                "total_events": 1,
+                "keywords": ["testuser", "testrepo", "en", "PushEvent"]
+            }
+        }
+        
+        frontmatter = builder._generate_frontmatter(digest)
+        data = load_frontmatter_yaml(frontmatter)
+        
+        # Check Open Graph metadata
+        assert "og" in data
```

Ruff (0.12.2)

Use of `assert` detected

(S101)